lighthouse-core/audits/autocomplete.js

```diff
@@ -43,30 +185,83 @@ class AutocompleteAudit extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.FormInput} input
+   * @return {{isValid: Boolean, orderWarn?: Boolean}}
+   */
+  static isValidAutocomplete(input) {
+    if (!input.autocomplete.attribute) return {isValid: false};
+    if (input.autocomplete.attribute.includes(' ') ) {
+      const tokenArray = input.autocomplete.attribute.split(' ');
+      for (const token of tokenArray) {
+        if (token.slice(0, 8) === 'section-') continue;
+        if (validAutocompleteTokens.includes(token)) continue;
+        return {isValid: false};
+      }
+      if (!input.autocomplete.property) return {isValid: false, orderWarn: true};
+      return {isValid: true};
+    }
+    return {isValid: validAutocompleteTokens.includes(input.autocomplete.attribute)};
+  }
+
   /**
    * @param {LH.Artifacts} artifacts
    * @return {LH.Audit.Product}
    */
   static audit(artifacts) {
     const forms = artifacts.FormElements;
```

Comment:
just thinking through our cases... 

https://docs.google.com/spreadsheets/d/1bFAD0HtcejpVPpp44r8WKK8KZcfwpTLA8gXQjB68c5I/edit#gid=0


more coming on this.. i'm thinking we may want to tweak the logic flow within this audit but it'll take me a lil longer..